### DISCUSSION TITLE
Italian translation of clone.js in remote level

### DISCUSSION THREAD

**Discussion by egiantu** [pcottle/learnGitBranching#885] [General]
Hi,
I'm new of git and github (I worked tih CVS, SVN ans Mercurial), and I use LearnGitBrnaching for learn.

I would translate the remain levels (not translted) of italian language.

I started with clone.js. I clone repository, modify clone.js and then use
**git add clone.js** 
**git commit -am "italian translation of clone.js in remote level"**

**git push**
but I received
__fatal: The current branch remote_clone_italian_translation has no upstream branch.
To push the current branch and set the remote as upstream, use
    git push --set-upstream origin remote_clone_italian_translation_"_

then I tried:
**git push --set-upstream origin remote_clone_italian_translation**

and received
_remote: Permission to pcottle/learnGitBranching.git denied to egiantu.
fatal: unable to access 'https://github.com/pcottle/learnGitBranching/': The requested URL returned error: 403_

how can i get the permissions ?

Thank you

**Comment by pcottle** [comment]
Hi and thanks for stopping by! It looks like you cloned my repository and then tried to push to a branch inside of it, which is why you got the permissions error.

I would instead fork the repo:
https://docs.github.com/en/get-started/quickstart/fork-a-repo

then push a branch to your fork, and then open up a pull request to merge the changes to my repo. there are some good tutorials out there that explain this in more detail

**Reply by egiantu** [reply]
Sorry @pcottle, as mentioned I take my first steps on Git and GitHub world.

Now, I fork repository and I send pull request.
